fifa19/www/style.css

```diff
@@ -2,6 +2,46 @@
   background: #c6deff;
 }
 
+a:has(#appsilon-logo) {
+  display: flex;
+}
+
+#appsilon-logo {
+  width: 110px;
+  cursor: pointer;
+  transition: transform 0.3s;
+}
+
+#appsilon-logo:hover {
```

Comment:
could you add a different color to highlight the active tab selected in the header?